vue/src/alerting/NotifChannelSlackForm.vue

```diff
@@ -28,19 +31,97 @@
       </v-row>
 
       <v-row>
-        <v-col>
-          <v-text-field
-            v-model="channel.params.webhookUrl"
-            label="Slack webhook URL"
-            placeholder="https://hooks.slack.com/services/********"
-            outlined
-            dense
-            hide-details="auto"
-            :rules="rules.webhookUrl"
-          />
+        <v-col cols="12">
+          <v-btn-toggle
+            v-model="authMethodIndex"
```

Comment:
It should be possible to use `v-model="channel.params.authMethod"` here directly if you change v-btn value accordingly:

```diff
--<v-btn value="0" class="text-none">
++<v-btn value="webhook" class="text-none">
```